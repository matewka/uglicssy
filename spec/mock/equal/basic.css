.a {
  margin-top: 10px;
}

.b {
  margin: 0 -15px;
}

.c,
.d {
  margin: 0 15px;
}

.c {
  width: 33.3333%;
}

.d {
  width: 41.6667%;
}