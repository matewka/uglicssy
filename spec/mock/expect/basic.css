.container {
    margin-top: 10px;
}

.row {
    margin: 0 -15px;
}
.col-sm-4, .col-sm-5 {
    margin: 0 15px;
}

.col-sm-4 {
    width: 33.3333%;
}

.col-sm-5 {
    width: 41.6667%;
}