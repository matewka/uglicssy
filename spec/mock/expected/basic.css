.a{margin-top:10px;}.b{margin:0 -15px;}.c,.d{margin:0 15px;}.c{width:33.3333%;}.d{width:41.6667%;}label.e{width:15%;}input.f[type="text"]{border-radius:2px;}