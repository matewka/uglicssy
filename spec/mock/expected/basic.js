//css
var asd = "a";

$('.other-class').hide();

$('.btn')
  .queue()
  //css
  .addClass("b")
  .show();

//css
$(".c").animate();

//css
$(".a").show();