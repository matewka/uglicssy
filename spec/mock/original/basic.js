//css
var asd = 'my-class';

$('.other-class').hide();

$('.btn')
  .queue()
  //css
  .addClass('warning')
  .show();

//css
$('.wtf').animate();

//css
$('.my-class').show();