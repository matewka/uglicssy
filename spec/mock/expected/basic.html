<html><head></head><body><div class="a">
    <div class="b">
        <div class="c"></div>
        <div class="d"></div>
        <div class="d"></div>
        <div class="c"></div>
    </div>
</div></body></html>